src/Controllers/PoolProviderController.cs

```diff
@@ -376,17 +376,17 @@ public async Task<IActionResult> GetAgentRequestStatus([FromBody] AgentRequestSt
                 case "unscheduled":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Helix work item in job {correlationId} for agent {workItemId} is currently {workItemDetails.State} or does not exist."
+                        statusMessage = $"(Work has been submitted to the Helix API, but has not started yet) Helix work item in job {correlationId} for agent {workItemId} is currently {workItemDetails.State}"
                     });
                 case "waiting":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Helix work item in job {correlationId} for agent {workItemId} is currently waiting for a machine."
+                        statusMessage = $"(Work has been submitted to the Helix API, and is waiting for a machine) Helix work item in job {correlationId} for agent {workItemId} is currently waiting for a machine."
                     });
                 case "failed":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Helix work item in job {correlationId} for agent {workItemId} failed.  Please check the logs."
+                        statusMessage = $"(Error state: Please contact dnceng with Job Id) Helix work item in job {correlationId} for agent {workItemId} failed.  Please check the logs."
```

Comment:
nit: Oh nos❕ Extra space between sentences 😈

Reply:
That's only been the grammar rule for a couple years but no problem :)